azure-pipelines-official.yml

```diff
@@ -37,6 +38,19 @@ stages:
     - powershell: Write-Host "##vso[task.setvariable variable=SourceBranchName]$('$(Build.SourceBranch)'.Substring('refs/heads/'.Length))"
       displayName: Setting SourceBranchName variable
 
+    - powershell: |
+        Write-Host "Setting up the build for PR validation by merging refs/pull/$(PrNumber)/merge into $(SourceBranchName)."
+        if ($SourceBranchName -notlike '*-vs-deps') {
+          Write-Host  "##vso[task.LogIssue type=warning;]The base branch for insertion validation is $(SourceBranchName), which is not a vs-deps branch."
+        }
+        git pull origin refs/pull/$(PrNumber)/merge
```

Comment:
In this context, does `git pull` perform a merge?

Reply:
I think it does. See https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/results?buildId=3518110